### PR TITLE
FOGL-2413 - Removed DataRetrievalError

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

File: python/foglamp/plugins/south/systeminfo/systeminfo.py [hunk 1]
```diff
@@ -218,13 +216,11 @@ def insert_reading(asset, time_stamp, data):
     try:
         time_stamp = utils.local_timestamp()
         get_system_info(time_stamp)
-    except asyncio.CancelledError :
-        pass
     except OSError as ex:
         _LOGGER.exception("Encountered System Error: {}".format(str(ex)))
```
Comment (praveen-garg) [inline python/foglamp/plugins/south/systeminfo/systeminfo.py]: why there is no raise if OSError happens

**Review by praveen-garg — CHANGES_REQUESTED** [review]

File: python/foglamp/plugins/south/systeminfo/systeminfo.py (hunk 1 above)
Comment (amarendra-dianomic) [reply python/foglamp/plugins/south/systeminfo/systeminfo.py]: done

File: python/foglamp/plugins/south/systeminfo/systeminfo.py [hunk 2]
```diff
@@ -218,13 +216,9 @@ def insert_reading(asset, time_stamp, data):
     try:
         time_stamp = utils.local_timestamp()
         get_system_info(time_stamp)
-    except asyncio.CancelledError :
-        pass
```
Comment (praveen-garg) [inline python/foglamp/plugins/south/systeminfo/systeminfo.py:222]: why this has been removed? we should keep it.

**Review by praveen-garg — CHANGES_REQUESTED** [review]

File: python/foglamp/plugins/south/systeminfo/systeminfo.py (hunk 2 above)
Comment (amarendra-dianomic) [reply python/foglamp/plugins/south/systeminfo/systeminfo.py:222]: asyncio is no longer in use as this plugin has now become a poll plugin, handled via South C server.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**